app/models/manageiq/providers/azure/inventory/collector/network_manager.rb

```diff
@@ -10,4 +10,8 @@ def security_groups
   def load_balancers
     @load_balancers ||= gather_data_for_this_region(@lbs)
   end
+
+  def network_routers
```

Comment:
if we want to ignore targeted refresh, in Collector::TargetCollection, we should have

```
def network_routers
  [] # TODO add targeted refresh
end
```

then the specs should pass, otherwise it calls the method in the Collector super class